text/trace/0136-error_flagging.md

```diff
@@ -0,0 +1,89 @@
+# Error Flagging with Status Codes
+
+This proposal adds two status codes explicitly for use as overrides by the end user, and proposes a canonical mapping of semantic conventions to status codes. This clarifies how error reporting should work in OpenTelemetry.
```

Comment:
It's actually three status codes if "`UNSET` is the default status code" is included. It's also not only intended for overriding by the end user but also to be set by instrumentations.

Reply:
fixed